test/data/Cardano/Wallet/Api/ApiWallet.json

```diff
@@ -1,110 +1,116 @@
 {
-    "seed": 1080154664973591237,
+    "seed": -7193288254086185690,
     "samples": [
         {
             "passphrase": {
-                "last_updated_at": "1864-04-24T03:49:30.165952219949Z"
+                "last_updated_at": "1864-06-06T17:04:22.128113480808Z"
```

Comment:
why here year 1864?

Reply:
That is arbitrarily generated :man_shrugging: